src/main/java/Rei.java

```diff
@@ -0,0 +1,168 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.List;
+
+public class Rei {
```

Comment:
I think it'd be good to have a method/static constant to represent line breaks and other things. I put mine in the Ui class cause I think it makes the most sense, but anywhere's fine, probably. Here's what I mean
```
public void printLine() {
        System.out.println("____________________________________________________________");
    }
```